_includes/configuration/images-dataimg-filter.md

```diff
@@ -1,6 +1,8 @@
 ## images_dataimg_filter
 
-The **images_dataimg_filter** option lets you specify a filter predicate function for disabling the logic that converts base64 images into blobs while within the editor. This feature is generally used for very advanced integrations where you handle all base64/blob images with custom logic.
+> **Important**: This option was deprecated with the release of {{site.productname}} 5.3. `images_dataimg_filter` may be removed in a future release of {{site.productname}}.
+
+The **images_dataimg_filter** option is used to filter `<img>` elements before they are passed to [`image_upload_handler`]({{site.baseurl}}/configure/file-image-upload/#images_upload_handler).
```

Comment:
Not sure we should be removing the blob details here, as no doubt someone will complain about that?

Reply:
Also, should we keep the part about this being useful for advanced integrations? It's vague, but helps to keep people away from using it.

Reply:
Discussed on slack and we're fine with dropping that part since it's very vague.